### DISCUSSION TITLE
How advanced is POS Tagging?

### DISCUSSION THREAD

**Discussion by ryanj11** [nlp-compromise/nlp_compromise#898] [Q&A] [answered]
Hi!

How advanced is the POS tagging in Compromise?

With this test sentence - "We will grant you your wish." These are the tags it finds for the word "grant" -
`tags: [
   'MaleName',
   'FirstName',
   'Person',
   'ProperNoun',
   'Singular',
   'Noun'
]`

Grant is a verb in this sentence but Compromise thinks the sentence is about a guy called Grant, even though the g in grant isn't capitalized.

**Comment by spencermountain** [comment, marked as answer]
hey @ryanj11 yeah, that's a good example. 
you can track-down the reasoning for each tag with `nlp.verbose(true)`

in this case, it passes straight-through our [person-name tagging rules](https://github.com/spencermountain/compromise/blob/master/src/02-tagger/04-correction/matches/10-people.js) which is a bummer. Grant should be added to [this list](https://github.com/spencermountain/compromise/blob/master/src/02-tagger/04-correction/_ambig.js#L30)

A lot of this is improving in v14. I just checked and grant is a verb in that sentence. That's a week or two off.
cheers
